Tests/MapboxMapsTests/Style/StyleIntegrationTests.swift

```diff
@@ -276,4 +276,33 @@ internal class StyleIntegrationTests: MapViewIntegrationTestCase {
         XCTAssertTrue(sourceTerrainProperty is NSNull)
         XCTAssertTrue(exaggerationTerrainProperty is NSNull)
     }
+
+    func testCustomVectorSource() {
+        // add GeoJSON source to map style
+        let sourceId = "source"
+        var source = GeoJSONSource()
+        source.data = .empty
+        try! self.style.addSource(source, id: sourceId)
+
+        // style sources’ identifiers count increases to 1, excluding custom vector sources
+        XCTAssertFalse(self.style.allSourceIdentifiers is NSNull)
+        XCTAssertEqual(self.style.allSourceIdentifiers.count, 1)
```

Comment:
For a more thorough test, check the whole array:

```suggestion
        XCTAssertEqual(self.style.allSourceIdentifiers, [sourceId])
```

Reply:
ah thank you, this makes more sense